<launch>
  <arg name="data_path" default="/media/lixin/7A255A482B58BC84/lx/CARLA_Shipping_my-0.9.13-3-g0aba485f0/LinuxNoEditor/PythonAPI/DataGenerator/data/test/"/>
  <arg name="dataset" default="training"/>22-05-22
  <arg name="frame_size" default="0"/>
  <arg name="current_frame" default="0"/>

  <!-- Object visualizer -->
  <node pkg="kitti_visualizer" type="object_visualizer_node" name="object_visualizer_node" output="screen" required="true">
    <param name="data_path" type="str" value="$(arg data_path)" />
    <param name="dataset" type="str" value="$(arg dataset)" />
    <param name="frame_size" type="int" value="$(arg frame_size)" />
    <param name="current_frame" type="int" value="$(arg current_frame)" />
    <param name="save_image" type="bool" value="false" />
    <param name="show_depth_cloud" type="bool" value="false" />
    <param name="save_stereo_cloud" type="bool" value="false" />
  </node>

  <!-- Rviz -->
  <node pkg="rviz" name="rviz" type="rviz" args="-d $(find kitti_visualizer)/rviz/object_visualizer.rviz" />

</launch>
